src/components/TextBubble/TextBubbleService.tsx

```
@@ -21,20 +18,18 @@ export function TextBubbleService() {
         <div className="body2 shadow-bubble max-w-[240px] rounded-[2px_12px_12px_12px] bg-white px-[10px] py-[10px] break-words whitespace-pre-wrap">
           <div
             className="whitespace-pre-wrap"
-            dangerouslySetInnerHTML={{ __html: MOCK_ORIGINAL }}
+            dangerouslySetInnerHTML={{ __html: normal }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)